Dockerfile

```diff
@@ -22,11 +22,22 @@ COPY --from=jenkinsfilerunner-mvncache /mavenrepo /mavenrepo
 ADD . /jenkinsfile-runner
 RUN cd /jenkinsfile-runner && mvn package
 
-FROM jenkins/jenkins:${JENKINS_VERSION}
+FROM jenkins/jenkins:${JENKINS_VERSION} as jenkins
 USER root
-RUN mkdir /app && unzip /usr/share/jenkins/jenkins.war -d /app/jenkins
+# Delete big files not needed
+RUN mkdir /app && unzip /usr/share/jenkins/jenkins.war -d /app/jenkins && \
+  rm -rf /app/jenkins/scripts /app/jenkins/jsbundles /app/jenkins/css /app/jenkins/images /app/jenkins/help /app/jenkins/WEB-INF/detached-plugins /app/jenkins/winstone.jar /app/jenkins/WEB-INF/jenkins-cli.jar /app/jenkins/WEB-INF/lib/jna-4.5.2.jar
```

Comment:
What happens if a plugin relies on a detached plugin?

Reply:
the install-plugins script would install it

Reply:
I am not sure about that: https://github.com/jenkinsci/docker/blob/master/install-plugins.sh#L148

Reply:
there's no war in the last dockerfile `FROM` step where install-plugins is run